scripts/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import("eslint").Linter.Config} */
```

Comment:
Is this file needed?

Reply:
I realized that it was giving ide errors because the folder had files with console.logs but the root eslint config doesn't allow console.logs.

But scripts folder contains files for cli scripts.

So i suppose it's needed.

Reply:
Hmm but this way this file is also not used by anything.... I think the real solution would be to enforce root eslint config for every single file (right now it's only for code that is actually published) and then ignore valid console.logs with eslint ignore comments. I'd remove this from this PR and do this in a separate one.